05-visualisation-ggplot2.Rmd

```diff
@@ -185,6 +185,7 @@ sections and points are counted wihin hexagons. The number of points per hexagon
 is encoded by color.
 
 ```{r}
+library(hexbin)
```

Comment:
I agree. I think here we could use a different color palette to not have to call another package.